### DISCUSSION TITLE
Two additional costs

### DISCUSSION THREAD

**Discussion by Floppe** [custom-components/nordpool#321] [Q&A] [answered]
Hi!

Would it be possible to have 2 additional costs, so I could get 2 graphs. One for buying and one for selling. Here in Finland we have different margins and different VAT for consumption vs production.

**Comment by Hellowlol** [comment, marked as answer]
No, that will not be supported but you could add additional sensors so you can combine them in some visual presentation.

**Comment by Floppe** [comment]
Yeah, thanks. Guess I could create two new sensors that goes through attributes and the raw data and fixes the arrays.

**Reply by Hellowlol** [reply]
Just add two sensors with two different additional costs.

**Reply by Floppe** [reply]
Ah.. I could not get it to work with YAML.. but tested with UI and thanks.
Is it possible with YAML also? So one can see what template was used.

**Reply by Hellowlol** [reply]
Yes it’s possible. Just remember that they can not have identical setttings. Set one sensor to use one additional precision
